preferences.json

```diff
@@ -3,8 +3,8 @@
     "windowWidth" : 1435.0,
     "windowHeight" : 813.0,
     "windowCoordinates" : {
-      "x" : -4,
-      "y" : -4
+      "x" : 164,
```

Comment:
Oddly, my own MainApp opens with odd sizes sometimes. Was something changed along the way?

Reply:
ahh it's probably because of resizing. because preferences.json was taken out of gitignore, now everytime it is rewritten